internal/application/workload.go

```diff
@@ -35,24 +35,6 @@ func (a *Workload) Delete(ctx context.Context, gitea GiteaInterface) error {
 		return pkgerrors.Wrap(err, "failed to delete repository")
 	}
```

Comment:
If a workload is a specific `Deployment`, then deleting the git repo is not part of deleting the workload. In any case, all that's left now is the repo deletion which makes the method name even less appropriate. Anyhow, leave it for the upcoming refactoring.

Reply:
Yes, I'll change that with my current story. I have to change the delete code anyhow.